schema/retries.json

```diff
@@ -40,9 +40,9 @@
         },
         "jitter": {
           "type": ["number","string"],
-          "minimum": 0.0,
+          "minimum": 0,
           "maximum": 1.0,
```

Comment:
its ok to make this a "1" here as well?

Reply:
Actually, we need to have `1.0`, but 0.. :-1:

Reply:
It worked only for integers everywhere (regarding the minimum, maximum fields, not the type).
This PR also complained for the `workflow.json` file for some change in some min/max field.

Reply:
@iwita can you please paste the output of the JSON schema tool, please?

Reply:
There it is:

`gojsonschema: Loading /tmp/serverlessworkflow/schema/common.json`
`gojsonschema: Loading /tmp/serverlessworkflow/schema/events.json`
`gojsonschema: Warning: Multiple types map to the name "Metadata"; declaring duplicate as "Metadata_1" instead`
`gojsonschema: Warning: Object type with no properties has required fields; skipping validation code for them since we don't know their types`
`gojsonschema: Loading /tmp/serverlessworkflow/schema/functions.json`
`gojsonschema: Warning: Object type with no properties has required fields; skipping validation code for them since we don't know their types`
`gojsonschema: Loading /tmp/serverlessworkflow/schema/workflow.json`
`gojsonschema: Failed: json: cannot unmarshal number 0.0 into Go struct field Type.definitions.properties.minimum of type int`
`sed: can't read zz_generated.types_workflow.go: No such file or directory`
`mv: cannot stat 'zz_generated.types_*.go': No such file or directory`
`'./hack/zz_generated.types_state_impl.go.template' -> './model/zz_generated.types_state_impl.go'`
`model/zz_generated.types_state_impl.go`

Reply:
But this PR is fixing the `0.0` value. have you used this PR to generate the fields? :thinking:

Reply:
You are right. I forgot to checkout:

`gojsonschema: Warning: Object type with no properties has required fields; skipping validation code for them since we don't know their types`
`gojsonschema: Loading /tmp/serverlessworkflow/schema/retries.json`
`gojsonschema: Failed: json: cannot unmarshal number 1.0 into Go struct field Type.definitions.properties.minimum of type int`

Reply:
This is so weird. So `maximum` has to be `1.0` and minimum must be `1`.